### DISCUSSION TITLE
Add `craft resave/addresses` to the console

### DISCUSSION THREAD

**Discussion by adampatpattison** [craftcms/cms#13720] [Ideas] [answered]
I've noticed that we don't have a `resave/addresses` option in the console commands. 

I've written my own extension in the project I'm doing as I'm automatically generating the Labels on save and want to re-save them all. 

Might be a quick win to add this into the core resave controller.

**Comment by brandonkelly** [comment, marked as answer]
Good call! Just added this for the next release (df09cb5c66b4826308cf52727a001447ade661a8).

**Reply by brandonkelly** [reply]
Craft 4.5.6 is out with that new command.
